src/components/interfaces/HMI_API.xml

```diff
@@ -1253,20 +1253,20 @@
   </element>
 </enum>
 
-<enum name="SteeringWheelLocation">
-  <description>Describes the location of the Steering Wheel.</description>
-  <element name="LEFT" />
-  <element name="RIGHT" />
-  <element name="CENTER" />
-</enum>
-
 <enum name="DeliveryMode">
   <description>The mode in which the SendLocation request is sent</description>
   <element name="PROMPT" />
   <element name="DESTINATION" />
   <element name="QUEUE" />
 </enum>
 
+<enum name="SteeringWheelLocation">
```

Comment:
@LuxoftAKutsan done in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215